androidApp/src/main/java/dev/johnoreilly/confetti/splash/SplashView.kt

```diff
@@ -20,7 +21,11 @@ fun SplashRoute(
     val conference by viewModel.conference.collectAsStateWithLifecycle()
     when (val conference1 = conference) {
         null -> LoadingView()
-        CONFERENCE_NOT_SET -> navigateToConferences(ConferencesKey)
+        CONFERENCE_NOT_SET -> {
+            SideEffect {
+                navigateToConferences(ConferencesKey)
+            }
+        }
         else -> navigateToSessions(SessionsKey(conference1))
```

Comment:
Probably also for navigateToSessions. 

Generally my mental model is SideEffect can run after the UI is drawn. And tell the compose runtime, this shouldn't be skipped.

Reply:
Makes sense 👍